source/_integrations/zha.markdown

```diff
@@ -253,14 +273,25 @@ zha:
   zigpy_config:
     ota:
       ikea_provider: true                        # Auto update Trådfri devices
+      inovelli_provider: true                    # Auto update INOVELLI devices
       ledvance_provider: true                    # Auto update LEDVANCE/OSRAM devices
       salus_provider: true                       # Auto update SALUS/Computime devices
-      inovelli_provider: true                    # Auto update INOVELLI devices
+      sonoff_provider: true                      # Auto update Sonoff (ITead) devices
       thirdreality_provider: true                # Auto update 3REALITY devices
       #otau_directory: /path/to/your/ota/folder  # Utilize .ota files to update everything else
 ```
 
-You can choose if the IKEA, LEDVANCE, SALUS, INOVELLI or THIRDREALITY provider should be set to enabled (`true`) or disabled (`false`) individually. After the OTA firmware upgrades are finished, you can set these to `false` again if you do not want ZHA to automatically download and perform OTA firmware upgrades in the future.
+To use generic OTA providers for zigpy in the ZHA integration you need to provide URLs to your generated JSON file(s) using the following configuration to your `configuration.yaml` and restart Home Assistant:
```

Comment:
Remote providers are for development and testing, they do not belong in ZHA docs/

Reply:
OK removed